.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
[OPTIONAL]
- Add to .gitignore directories that aren't necessary for your project.